allensdk/brain_observatory/ecephys/ecephys_session.py

```diff
@@ -144,6 +144,7 @@ def stimulus_names(self):
     def __init__(self, api, **kwargs):
         self.api: EcephysApi  = api
 
+        self.session_id = self.LazyProperty(self.api.get_session_id)
```

Comment:
Actually, I have a question about this: are you always handing this Session constructor an API that has been initialized to a specific id?

Reply:
That was my plan - it seems like for a session-level api to access data at all it can be reasonably expected to be able to access the session id. This would not be true of a dataset-level api.

This highlights a need for me to rename EcephysApi -> EcephysSessionApi